core/chain_manager.go

```diff
@@ -800,8 +797,18 @@ func (self *ChainManager) merge(oldBlock, newBlock *types.Block) error {
 		PutTransactions(self.chainDb, block, block.Transactions())
 		PutReceipts(self.chainDb, GetBlockReceipts(self.chainDb, block.Hash()))
 
+		addedTxs = append(addedTxs, block.Transactions()...)
+	}
+
+	var diff types.Transactions
+	types.TxDifference(deletedTxs, addedTxs)
```

Comment:
Thanks. This changes from **modify target** to single function. I didn't update to the new format. Good catch @Gustav-Simonsson